### ISSUE TITLE
'Quick Play' main menu option

### ISSUE THREAD

**Issue by portaljumper007** [PirateGame/Pirate-Game-Frontend#14]
Try matchmaking with another client in a quick play waiting room, and if there is no one waiting then set up a new game with one AI player and join them to that and start it.

**Comment by Alex40144** [comment]
should there be like a 5 min timeout? to make sure people actually get matched with another player?

should we put an AI tag on ai?

**Comment by Alex40144** [comment]
we need to have a think about how this is going to work, public games probably.

**Comment by portaljumper007** [comment]
Yes, that functionality is already implemented as the "quickplay" tag that is associated when a game is made and is modifiable. should be pretty easy to implement through to the website.

**Comment by Alex40144** [comment]
so we can get a list of "public" games and return them with stats like grid size and player count?

**Comment by portaljumper007** [comment]
Yes, absolutely, also i can easily write a "random join" function

**Comment by Alex40144** [comment]
Just give the player a list of open/public games, and they can choose.

**Comment by portaljumper007** [comment]
> Just give the player a list of open/public games, and they can choose.

Yes, that's purely website functionality really though - I'll write a function to return joinable game names for you. There could also be a random join button, which i've already coded for (joinLobby with no specified game name, it will return the game name it chose)

**Comment by portaljumper007** [comment]
"listQuickPlayGames()" (now implemented) ^
